mixer/cmd/build.go

```diff
@@ -568,3 +619,39 @@ func init() {
 		append(externalDeps[buildUpdateCmd],
 			externalDeps[buildImageCmd]...)...)
 }
+
+func checkFormatMismatch(b *builder.Builder) error {
+	//Skip check if offline
+	if builder.Offline {
+		return nil
+	}
+
+	hostFormat, upstreamFormat, err := b.GetHostAndUpstreamFormats()
+	if err != nil {
+		return err
+	}
+
+	if hostFormat == "" {
+		fmt.Println("Warning: Unable to determine host format. Running natively may fail.")
+	} else if hostFormat != upstreamFormat {
+		fmt.Println("Warning: The host format and mix upstream format do not match.",
```

Comment:
We should `log` these so they go to `stderr` like other warnings.

Reply:
Indeed! I wrote this code before the log change, I let this one slip. Good catch!